examples/07.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+
+import elabapi_python
+from elabapi_python.rest import ApiException
+
+#replace with your api key
+my_api_key = 'xxxxx'
```

Comment:
Use the same header as other examples, which includes the default `dev` key so the script can be executed locally without changes.